Dockerfile_archive

```diff
@@ -12,28 +12,32 @@ RUN apt-get update --yes \
         iproute2 \
  && apt-get upgrade --yes \
  && apt-get satisfy --yes --no-install-recommends \
+       ## Standard packages
```

Comment:
What do "standard" and "extra" mean here?

Reply:
See https://github.com/jelmer/janitor/pull/929